rusk-wallet/README.md

```diff
@@ -18,6 +18,8 @@ OPTIONS:
                                        [default: uds]
     -s, --socket-path <SOCKET_PATH>    Path for setting up the unix domain socket [default: /tmp/
                                        rusk_listener]
+    -s, --skip-recovery                Skip wallet recovery phrase (useful for headless wallet
```

Comment:
If I use `-s` will be interpreted as "socket-path" or "skip-recovery"?

Reply:
Great catch indeed 🚀